Formula/voltdb.rb

```diff
@@ -17,11 +17,15 @@ class Voltdb < Formula
   def install
     system "ant"
 
-    inreplace Dir["bin/*"] - ["bin/voltadmin","bin/voltdb","bin/rabbitmqloader"],
-      /VOLTDB_LIB=\$VOLTDB_HOME\/lib/, "VOLTDB_LIB=$VOLTDB_HOME/lib/voltdb"
+    inreplace Dir["bin/*"] - ["bin/voltadmin", "bin/voltdb", "bin/rabbitmqloader"],
+      %r{VOLTDB_LIB=\$VOLTDB_HOME\/lib}, "VOLTDB_LIB=$VOLTDB_HOME/lib/voltdb"
 
     (lib/"voltdb").install Dir["lib/*"]
     lib.install_symlink lib/"voltdb/python"
     prefix.install "bin", "tools", "voltdb", "version.txt", "doc"
   end
+
+  test do
+    assert_match version.to_s, shell_output("voltdb --version")
```

Comment:
Just FYI, in future, calling an executable that lives inside the Cellar should _always_ be done with a full path, i.e. `#{bin}/voltdb --version`